user-interface/src/case-detail/panels/CaseDetailAuditHistory.test.tsx

```diff
@@ -133,7 +133,7 @@ describe('audit history tests', () => {
     render(<CaseDetailAuditHistory caseId={caseId} />);
 
     const emptyAssignments = await screen.findByTestId('empty-assignments-test-id');
-    expect(emptyAssignments).toHaveTextContent('There are no assignments in the case history.');
+    expect(emptyAssignments).toHaveTextContent('No changes have been made to this case');
```

Comment:
Was the period left off intentionally?
```suggestion
    expect(emptyAssignments).toHaveTextContent('No changes have been made to this case.');
```

Reply:
Yes.  I believe in general we are avoiding periods in things like tooltips, but I guess this isn't a tooltip so it deserves a period.